content/en/community/contributing/code/releasing/pyxform.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Releasing Pyxform"
+linkTitle: "Releasing Pyxform"
+weight: 5
+description: >
+    Instructions for updating medic/pyxform to new XLSForm/pyxform releases
+relatedContent: >
+    releases/
+aliases: >
+  /contribute/code/releasing/pyxform
```

Comment:
```suggestion
```

I don't think we need this. Usually aliases are for older pages that get moved around (and the aliases let us maintain backwards linkability to the old spot).